README.md

```diff
@@ -103,6 +114,12 @@ The next 4 bytes are called the "prefix bytes" (in square brackets).
 > <0xA8 0xFC 0x54> [0xBB 0x9C 9x83 9xDD]
 ```
 
+We reserve the last 3 bits for the typ3 of the concrete type, so in
+this case the final prefix bytes become `(0xDD & 0xF8) | <typ3-byte>`.
+The type byte for a struct is 0x03, so if the concrete type were a struct,
+the final prefix byte would be `0xDE`.
```

Comment:
Is this correct?

(0xDD & 0xF8) | <typ3-byte>:: (0xD8) | (0x03) = 0xDB

Reply:
yeah I think you're right.